app/src/main/java/com/github/sdp_begreen/begreen/models/TrashCategory.kt

```diff
@@ -0,0 +1,53 @@
+package com.github.sdp_begreen.begreen.models
+
+import android.os.Parcel
+import android.os.Parcelable
+import com.github.sdp_begreen.begreen.R
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+
+enum class TrashCategory(val id : String, val title : String, val color : Float) : Parcelable {
+
+    // The title will be displayed in the screen to the user, so we prefer to define it in the strings.xml instead
+    // of Capitalizing the toString() value.
+
+    // The id is unique (stored in the database). We preferred to store a category by its "id" instead of its "value"
+    // name (i.e PLASTIC) because the value name could be changed afterward.
+
+    // TODO : use the strings.xml file for the title field. Since the class is a model class, it doesn't recognize this file.
```

Comment:
I was not sure about this, but I thought that every string displayed on the screen should be in the strings.xml
The strings.xml automatically handles the Locality, so it avoids us from creating a method which returns the LocalizedString to display on the screen.

I've also noticed that the `Action` class doesn't use any LocalizedString. We can discuss it with the team later but it's not a big deal for now.

Regarding what you said :

> we previously just could add or remove a member of the enum

If we decide to remove it, it will be no more consistent with the backend. So I see it like another layer of security : if someone removes a member of the enum, he will be warned by the compiler.

Reply:
You are right, I will apply the change in the map fragment dropdown !